mkt/extensions/indexers.py

```diff
@@ -57,6 +57,7 @@ def get_mapping(cls):
                     },
                     # Name for sorting.
                     'name_sort': cls.string_not_analyzed(doc_values=True),
+                    'guid': cls.string_not_analyzed(),
```

Comment:
To be compatible with the way we store it for apps in ES. Not necessary right now but will help in the future (guid lookups are used in the lookup tool), so I figured why not.